internal/cli/atlas/quickstart/quick_start.go

```diff
@@ -0,0 +1,431 @@
+// Copyright 2020 MongoDB Inc
+//
+// Licensed under the Apache License, Version 2.0 (the "License");
+// you may not use this file except in compliance with the License.
+// You may obtain a copy of the License at
+//
+//      http://www.apache.org/licenses/LICENSE-2.0
+//
+// Unless required by applicable law or agreed to in writing, software
+// distributed under the License is distributed on an "AS IS" BASIS,
+// WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+// See the License for the specific language governing permissions and
+// limitations under the License.
+package quickstart
+
+import (
+	"crypto/rand"
+	"errors"
+	"fmt"
+	"math/big"
+	"os/exec"
+	"regexp"
+	"strings"
+
+	"github.com/AlecAivazis/survey/v2"
+	"github.com/mongodb/mongocli/internal/cli"
+	"github.com/mongodb/mongocli/internal/config"
+	"github.com/mongodb/mongocli/internal/convert"
+	"github.com/mongodb/mongocli/internal/flag"
+	"github.com/mongodb/mongocli/internal/net"
+	"github.com/mongodb/mongocli/internal/randgen"
+	"github.com/mongodb/mongocli/internal/store"
+	"github.com/mongodb/mongocli/internal/usage"
+	"github.com/spf13/cobra"
+	atlas "go.mongodb.org/atlas/mongodbatlas"
+)
+
+const quickstartTemplate = "Now you can connect to your Atlas cluster with: mongo -u %s -p %s %s\n"
+
+const (
+	replicaSet        = "REPLICASET"
+	diskSizeGB        = 20.0
```

Comment:
you can use, https://github.com/mongodb/go-client-mongodb-atlas/blob/943e1c852f6a1389f7281720566b71eaff148214/mongodbatlas/clusters.go#L174